Index: src/main/java/Main.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/Main.java	(revision 7b56176a42a27361ffd61f61946593b34e6ed4d5)
+++ src/main/java/Main.java	(date 1578302283168)
@@ -9,8 +9,8 @@
 
     public static void main(String[] args) {
         //  spausdintiStudentoInf();
-         uzkrautiDuomenis();
-//        kursoVidurkis(2018);
+        uzkrautiDuomenis();
+        kursoVidurkis(2018);
     }
 
     static void spausdintiStudentoInf() {
